sdk/quantum/quantum-jobs/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## 1.0.0-beta.2 (Unreleased)
 
+### Breaking Changes
+
+- Migrated to the Core v2 HTTP pipeline. As a result of this migration:
+  - The response types no longer contain the raw response `_response`.
```

Comment:
maybe a mention of `onResponse` being added to every operation options to facilitate retrieving this information

Reply:
Fixed